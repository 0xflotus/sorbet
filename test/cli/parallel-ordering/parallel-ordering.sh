#!/bin/bash

dir="test/cli/parallel-ordering"
main/ruby-typer $dir/1.rb $dir/2.rb $dir/3.rb 2>&1
