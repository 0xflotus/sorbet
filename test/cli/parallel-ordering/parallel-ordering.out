test/cli/parallel-ordering/2.rb:3: `hi`: Method redefined http://go/e/4010
     3 |    def hi
            ^^^^^^
    test/cli/parallel-ordering/1.rb:3: Previous definition
     3 |    def hi
            ^^^^^^

test/cli/parallel-ordering/3.rb:3: `hi`: Method redefined http://go/e/4010
     3 |    def hi
            ^^^^^^
    test/cli/parallel-ordering/2.rb:3: Previous definition
     3 |    def hi
            ^^^^^^
Errors: 2
